libs/api/mocks/src/domains/applications/factories.ts

```diff
@@ -35,7 +35,7 @@ export const externalData = factory<DataProviderResult>({
       data: () => ({
         email: faker.internet.email(),
         emailVerified: true,
-        mobilePhoneNumber: faker.phone.phoneNumber('###-####'),
+        mobilePhoneNumber: faker.helpers.fromRegExp('[0-9]{3}-[0-9]{4}'),
```

Comment:
Should we use an actual regex here to benefit from potential compile time validation?

```suggestion
        mobilePhoneNumber: faker.helpers.fromRegExp(/[0-9]{3}-[0-9]{4}/),
```